onboarding/geo/README.md

```diff
@@ -0,0 +1,334 @@
+
+- **Temporal Data:** Satellites capture images at regular intervals, providing a historical record of changes over time. This is essential for tracking trends, monitoring seasonal variations, and assessing long-term impacts.
+
+- **Geospatial Data:** Satellite imagery includes geospatial information, enabling precise georeferencing and mapping. This allows us to correlate imagery with specific geographic locations on Earth.
+
+- **Resolution:** Satellite imagery comes in various resolutions, from low-resolution global views to high-resolution images that can capture fine details on the Earth's surface.
+
+## An Introduction to Satellite Bands
+
+To make the most of satellite imagery, it's essential to understand the concept of satellite bands. Satellite bands refer to the specific regions of the electromagnetic spectrum that satellites capture and record. Each band provides unique information about the Earth's surface and atmosphere, making them essential for various geospatial analyses.
+
+### Understanding Satellite Bands
+
+Satellite sensors are equipped with detectors that measure electromagnetic radiation in different parts of the spectrum. Here's an overview of the key satellite bands and their characteristics:
+
+### 1. **Visible Bands:**
+
+   - **Blue Band:** Captures blue light (approximately 0.45-0.52 micrometers). Useful for detecting water bodies and distinguishing water from other land features.
+
+   - **Green Band:** Captures green light (approximately 0.53-0.59 micrometers). Helps identify healthy vegetation, as chlorophyll reflects green light.
+
+   - **Red Band:** Captures red light (approximately 0.63-0.69 micrometers). Useful for vegetation health assessment and land cover classification.
+
+### 2. **Infrared Bands:**
+
+   - **Near-Infrared (NIR) Band:** Captures near-infrared light (approximately 0.75-1.4 micrometers). Used for vegetation analysis, as healthy vegetation strongly reflects NIR light.
+
+   - **Shortwave Infrared (SWIR) Band:** Captures shortwave infrared light (approximately 1.4-3.0 micrometers). Helps detect moisture content in soils and rock types.
+
+### 3. **Thermal Infrared Bands:**
+
+   - **Thermal Infrared (TIR) Band:** Captures thermal radiation (approximately 8-15 micrometers). Measures temperature variations on the Earth's surface, used in applications like detecting wildfires and monitoring urban heat islands.
+
+### 4. **Microwave Bands:**
+
+   - **Microwave Bands:** Microwaves (wavelengths ranging from millimeters to meters) are used in radar imagery. Microwaves can penetrate clouds and provide information on surface roughness, soil moisture, and more.
+
+Understanding these satellite bands and their specific uses is crucial for interpreting satellite imagery. Depending on your geospatial analysis goals, you may need to work with data from multiple bands to extract meaningful insights.
+
+#### Useful Links
+
```

Comment:
What can be useful to add is how satellites capture the earth. They are either using a bush broom or a which broom sensor, see for more explanation https://www.nv5geospatialsoftware.com/Learn/Blogs/Blog-Details/ArtMID/10198/ArticleID/16262/Push-Broom-and-Whisk-Broom-Sensors

The notion of this is important as it will lead to a different size and shape of the data capture. 

What is also important to consider for data collection is the orbit of a satellite, this is explained in https://www.esa.int/Enabling_Support/Space_Transportation/Types_of_orbits.

Maybe you can make a markdown table for the most important satellites that you noted below, add the orbit they have and the type of sensor.